test/parallel/test-dgram-bind-shared-ports.js

```diff
@@ -64,12 +64,15 @@ if (cluster.isMaster) {
 
   socket1.bind({
     address: 'localhost',
-    port: common.PORT,
+    port: 0,
     exclusive: false
-  }, () => {
-    socket2.bind({ port: common.PORT + 1, exclusive: true }, () => {
+  }, common.mustCall(() => {
+    socket2.bind({
+      port: socket1.address().port + 1,
```

Comment:
Is there a reason that this port has to be the first port + 1? Couldn't it just be `port: 0`?